deploy.py

```diff
@@ -384,6 +384,39 @@ def build_opensk(self):
     self._check_invariants()
     self._build_app_or_example(is_example=False)
 
+  def build_bootloader(self):
+    """Builds the upgrade bootloader."""
+    props = SUPPORTED_BOARDS[self.args.board]
+    info("Building bootloader")
+    rust_flags = [
+        f"--remap-path-prefix={os.getcwd()}=",
+        "-C",
+        "link-arg=-Wl,-Tlink.x",
+        "-C",
+        "link-arg=-nostartfiles",
+    ]
+    env = os.environ.copy()
+    env["RUSTFLAGS"] = " ".join(rust_flags)
```

Comment:
I suggest to remove the flags from the `bootloader/.cargo/config` file to ensure they are only defined once.

Reply:
Good point. README is updated to match, also setting the RUSTFLAGS manually.